src/compositor/view.c

```diff
@@ -205,8 +207,8 @@ wlc_view_get_bounds(struct wlc_view *view, struct wlc_geometry *out_bounds, stru
       // xdg-surface client that draws drop shadows or other stuff.
       out_bounds->origin.x -= view->surface_commit.visible.origin.x;
       out_bounds->origin.y -= view->surface_commit.visible.origin.y;
-      out_bounds->size.w += surface->size.w - view->surface_commit.visible.size.w;
-      out_bounds->size.h += surface->size.h - view->surface_commit.visible.size.h;
+      out_bounds->size.w += surface->size.w / scale - view->surface_commit.visible.size.w;
+      out_bounds->size.h += surface->size.h / scale - view->surface_commit.visible.size.h;
```

Comment:
surface's size seems to be the geometry*buffer_scale , but the bounds' coordinate should be  same as geometry?